blazor/pdfviewer-2/server-deployment/deploy-maui-windows.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: Deploy SfPdfViewer in Blazor MAUI in windows | Syncfusion
+description: Checkout and learn here all about how to deploy SfPdfViewer in Blazor MAUI in windows in Syncfusion Blazor SfPdfViewer component and much more details.

```

Comment:
Description: Learn how to deploy SfPdfViewer in Blazor MAUI in Windows in Syncfusion Blazor SfPdfViewer component and much more details.